spec/requests/acuant_sdk_spec.rb

```diff
@@ -1,12 +1,10 @@
 require 'rails_helper'
 
 describe 'requesting acuant SDK assets' do
-  base_urls = ['/acuant/11.4.3', '/acuant/11.5.0']
+  base_urls = ['/acuant/11.5.0']
```

Comment:
This could be changed to a singular string `base_url`, but I expect we'll need to do this again with future upgrades, so may as well keep it this way to easily support multiple bases?

Reply:
On second thought, the behavior we're testing here needn't be tied to any specific version. In 57cefa5, I updated it to pick any version it finds within the `public/acuant` directory. This should help minimize the effort of future upgrades as well. It may become an issue if Acuant continues to change file names in future releases, but hopefully they're not doing this too often.